doc/reference/configuration/configuration_reference.rst

```diff
@@ -3220,6 +3220,26 @@ The ``lua`` section outlines the configuration parameters related to the Lua env
     | Default: 2147483648 (2GB)
     | Environment variable: TT_LUA_MEMORY
 
+.. _configuration_reference_lua_call:
+
+.. confval:: lua_call
+
+    Since version :doc:`3.3.0 </release/3.3.0>`, the ``lua_call`` option allows the specified user to perform the specified lua function on 
+    the instance during runtime.
```

Comment:
Perform the Lua function? I guess you tell about granting a permission to call it, not about the call itself.

Reply:
Seems solved.